PURL-TYPES.rst

```diff
@@ -170,15 +170,12 @@ conda
 
 cpan
 ----
-``cpan`` for CPAN Perl packages:
+``cpan`` for Perl package distributions published on CPAN:
 
 - The default repository is ``https://www.cpan.org/``.
-- The ``namespace``:
-  - To refer to a CPAN distribution name, the ``namespace`` MUST be present. In this case, the namespace is the CPAN id of the author/publisher. It MUST be written uppercase, followed by the distribution name in the ``name`` component. A distribution name MUST NOT contain the string ``::``.
-  - To refer to a CPAN module, the ``namespace`` MUST be absent. The module name MAY contain zero or more ``::`` strings, and the module name MUST NOT contain a ``-``
-
-- The ``name`` is the module or distribution name and is case sensitive.
-- The ``version`` is the module or distribution version.
+- The ``namespace`` is the CPAN id of the author/publisher. It MUST be written uppercase and is required.
+- The ``name`` is the distribution name and is case sensitive.
```

Comment:
Not sure, but would it make sense to still point out that…

> A distribution name MUST NOT contain the string ``::``.

…as earlier? I get it that it's strictly speaking not necessary, but as a hint for new users to understand the difference between a module name and a distribution name, I think it can be helpful.

(I have no strong opinions on this idea, and leave it to you to decide if this goes in or not)

Reply:
I agree, adding this sentence (`A distribution name MUST NOT contain the string ::`) reinforces the concept that only the distribution MUST be used.